### DISCUSSION TITLE
Cumulative plot across deployment restarts

### DISCUSSION THREAD

**Discussion by VictorCallejas** [prometheus/client_python#780] [Q&A]
Hello everyone,

I have set my grafana dashboard to see some custom metrics of my deployments.

The problem is when the deployment restarts, instead of adding up, the value gets restarted.

![image](https://user-images.githubusercontent.com/32225250/157203033-d39471ee-b913-4711-b0c0-fdae1a1b4808.png)

The query:
`watchdog_observer_files_processed_total{app=~"$watchdog"}`

Underlying this metric is a **Counter**.

Do you know what I can do so I got it cumulative across deployment restarts.

Thanks!

**Comment by csmarchbanks** [comment]
Sorry! I missed this discussion and converted your issue to a second discussion and answered it in https://github.com/prometheus/client_python/discussions/784#discussioncomment-2342154.
